crates/kubelet/src/volume/configmap.rs

```diff
@@ -0,0 +1,42 @@
+use std::path::PathBuf;
+
+use k8s_openapi::api::core::v1::{ConfigMap, KeyToPath};
+
+use super::*;
+
+pub(crate) async fn populate(
+    config_map: ConfigMap,
+    path: &PathBuf,
+    items: &Option<Vec<KeyToPath>>,
+) -> anyhow::Result<VolumeType> {
+    tokio::fs::create_dir_all(path).await?;
+    let binary_data = config_map.binary_data.unwrap_or_default();
+    let binary_data = binary_data.iter().map(|(key, data)| async move {
```

Comment:
Nit: We are consuming (and moving) the data here, so `into_iter` is a better fit